service/runtime/handler.go

```diff
@@ -199,16 +199,24 @@ func (h *Handler) ServeHTTP(rw http.ResponseWriter, inReq *http.Request) {
 		return
 	}
 
-	toUrl := fmt.Sprintf("%s%s", signerCfg.KeyConfig.BaseUrl, inReq.URL.Path)
-	h.log.LogF(" - To url '%s'\n", toUrl)
+	// toUrl := fmt.Sprintf("%s%s", signerCfg.KeyConfig.BaseUrl, inReq.URL.Path)
```

Comment:
Could you, please, remove this comment? It's public repo, so we should keep it clean

Reply:
agree, this shouldn't be here, somehow I missed it ;)